.travis.yml

```diff
@@ -21,6 +21,8 @@ install:
   - sudo apt-get -qq update
   - sudo apt-get install -y python3-coverage
 script:
+  # Disable internet access during unit tests.
+  - if [ "$TEST_SUITE" = unit ]; then echo 0 | sudo tee /proc/sys/net/ipv4/ip_forward; fi
```

Comment:
This needs to happen after installing the apt deps or we are not going to have those deps in :-)

Seems it can do the trick though